adev/shared-docs/components/viewers/example-viewer/example-viewer.component.spec.ts

```diff
@@ -252,6 +251,7 @@ describe('ExampleViewer', () => {
     const spy = spyOn(clipboardService, 'copy');
 
     await component.renderExample();
+    fixture.detectChanges();
```

Comment:
Since our tests are zoneless, we shouldn't have to call `detectChanges` explicitly. 
What's reason for invoking it here ?

Reply:
Looks like this also does it 
```suggestion
    await fixture.whenStable();
```

Reply:
I'm gonna be honest this whole PR was vibe coded 🤣